src/targeting/Find_targets.ml

```diff
@@ -206,6 +206,8 @@ type conf = {
   max_target_bytes : int;
   respect_gitignore : bool;
   respect_semgrepignore_files : bool;
+  (* Custom semgrepignore filename to use instead of '.semgrepignore' *)
```

Comment:
Pls remove this comment. We don't have comments in the other parameters and it's clear anyway.